doc/rados/operations/devices.rst

```diff
@@ -3,28 +3,32 @@
 Device Management
 =================
 
-Ceph tracks which hardware storage devices (e.g., HDDs, SSDs) are consumed by
-which daemons, and collects health metrics about those devices in order to
-provide tools to predict and/or automatically respond to hardware failure.
+Device management allows Ceph to address hardware failure. Ceph tracks hardware
+storage devices (for example, HDDs, SSDs) to see which devices are consumed by
```

Comment:
Suggest `devices (HDDs and SSDs) to see`
and
s/consumed/managed/

Reply:
What about NVMe as well?

Reply:
NVMe is an interface (like SAS or SATA), not a storage medium.  The odd stunt notwithstanding, NVMe implies SSD.